grakn-titan-factory/src/test/java/ai/grakn/factory/TitanTestBase.java

```diff
@@ -43,11 +42,6 @@ public static void setupMain(){
         titanGraphFactory = new TitanInternalFactory(TEST_SHARED, Grakn.IN_MEMORY, TEST_PROPERTIES);
     }
 
-    @AfterClass
```

Comment:
We need this. It may be deprecated but for our test environment we need it and we can't upgrade the embedded cass because of Titan